.gitignore

```diff
@@ -118,6 +118,8 @@ env/
 
 ### Configuration files ###
 .env
+.vagrant
 
 #geckodriver
 geckodriver.log
+geckodriver.exe
```

Comment:
Worth noting that because geckodriver.exe and geckodriver.log were already added to git, this doesn't remove them. You'll need to actively remove them with e.g. `git rm --cached geckodriver.exe` as well